semantic/semantic.fbs

```diff
@@ -155,24 +155,14 @@ table ArrayExpression {
 table FunctionExpression {
     loc:SourceLocation;
     defaults:ObjectExpression;
```

Comment:
I think in Rust the function expression does not have a `defaults` field, they are attached to the parameter instead.

Reply:
Also, don't you want to put `params:[FunctionParameter]` in here?

Reply:
Yeah, I rebased and some of my changes were dropped. I can fix that.